src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -113,11 +115,46 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
                             + "Please specify the location name and the player you want to share it with");
                     break;
                 }
-                if (!data.getSavedLocations().containsKey(args[1])) {
+                locationName = args[1];
+                String recipientName = args[2];
+                
+                if (!data.getSavedLocations().containsKey(locationName)) {
                     sender.sendMessage(ChatColor.RED + "No location with that name was found");
                     break;
                 }
+                Location sendLocation = data.getSavedLocations().get(locationName);
                 // TODO: Add share functionality
+                // Get the player we're sending to
+                Player recipient = org.bukkit.Bukkit.getPlayer(recipientName);
+                if (recipient == null) {
+                    sender.sendMessage(ChatColor.RED
+                            + "Player '" + recipientName + "' does not exist.");
+                    break;
+                }
+                if (recipientName.equals(player.getName())) {
+                    sender.sendMessage(ChatColor.RED
+                            + "You can't share a location with yourself!");
+                    break;
+                }
+                PlayerData recipData  = this.plugin.getDataManager().getData(recipient);
+                String shareLocName = player.getName() + " " + locationName;
+
+                if (recipData.getSavedLocations().containsKey(player.getName() + ":" + shareLocName)) {
+                    sender.sendMessage(ChatColor.RED
+                        + recipientName + " already has a location called "
+                        + shareLocName
+                    );
+                    break;
+                }
+
+                String locationString = "("
+                    + sendLocation.getWorld().getName() + " @ " + sendLocation.getBlockX()
+                    + ", " + sendLocation.getBlockY() + ", " + sendLocation.getBlockZ()
+                    + ")";
+
+                player.sendMessage(ChatColor.GREEN + String.format("Shared %s with %s", locationName, recipientName));
+                recipient.sendMessage(ChatColor.GREEN + String.format("%s has shared a location: %s (%s)", player.getName(), locationName, locationString));
+                recipData.getSavedLocations().put(player.getName() + ":" + locationName,sendLocation);              
```

Comment:
Fix formatting here

Reply:
What's wrong with the formatting?

Reply:
There are spaces at the end of the L157 that weren't stripped out.

Reply:
Can you clarify what you mean by L157?

Reply:
line 157